### PR TITLE
Add more i18n translations

### PR DESCRIPTION
**Pull request by Sawthis**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -25,6 +31,8 @@ api-rules:
   messages:
     create-failed: Failed to create the API Rule
     created: API Rule created
+    description: Expose Services outside the cluster with API Rules.
+    not-matching-results: Couldn't find API Rules matching this query.
     updated: API Rule updated
   services:
     error-load: Couldn't load service list
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:38]: Is this the "Kubernetes concept" kind of Service (capitalized) or the "Service Management" kind of service?
If Kubernetes Service, we need: 

```suggestion
    error-load: Couldn't load Service list
```

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -58,26 +83,34 @@ common:
...
+    loading: Loading...
     name: Name
+    resources: Resources
     status: Status
   labels:
     hostname: Hostname
     name: Name
     service: Service
+  messages:
+    doesnt-exist: doesn't exist.
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:107]: How does this look on the UI? :flushed: 
Do we tell users what it is that doesn't exist?  
Like `{{something}} doesn't exist.` ?

Comment (Sawthis) [reply core/src/i18n/en.yaml:107]: yes
`{entryType} "{entryId}" {t('common.messages.loading-name')}`

File: core/src/i18n/pl.yaml [hunk 3]
```diff
@@ -220,6 +266,16 @@ namespaces:
...
     memory-examples: Użyj zwykłej wartości w bajtach (128974848) lub odpowiednich przyrostków (takich jak 129e6, 129M, 123Mi).
     total-memory-limit: Zdefiniuj ograniczenia, które ograniczają całkowite zużycie pamięci w Twojej przestrzeni nazw.
+no-permissions:
+  title: Brak uprawnień
+node-details:
+  internal-ip: Wewnętrzny adres IP
+  involved-object: Zaangażowany obiekt
+  message: Wiadomość
+  name: Przegląd klastra — węzły
+  pod-cidr: Pod CIDR
```
Comment (Wawrzyn321) [inline core/src/i18n/pl.yaml:276]:
```suggestion
  pod-cidr: Adres CIDR Poda
```

File: core/src/i18n/pl.yaml [hunk 4]
```diff
@@ -220,6 +266,16 @@ namespaces:
     create: Wybierz tę opcję, aby wyłączyć istio, aby pośredniczyć w całej komunikacji między podami w Twojej przestrzeni nazw.
     memory-examples: Użyj zwykłej wartości w bajtach (128974848) lub odpowiednich przyrostków (takich jak 129e6, 129M, 123Mi).
     total-memory-limit: Zdefiniuj ograniczenia, które ograniczają całkowite zużycie pamięci w Twojej przestrzeni nazw.
+no-permissions:
+  title: Brak uprawnień
+node-details:
+  internal-ip: Wewnętrzny adres IP
+  involved-object: Zaangażowany obiekt
```
Comment (Wawrzyn321) [inline core/src/i18n/pl.yaml:273]:
```suggestion
  involved-object: Związany obiekt
```

File: core/src/i18n/pl.yaml [hunk 5]
```diff
@@ -200,6 +232,20 @@ jobs:
...
+  architecture-cpus: Architektura i procesory
+  cpu-m: Procesor (m)
+  cpu-usage: 'Zużycie procesora:'
+  cpus: Procesorów
+  gib: GiB
+  kube-proxy-version: Wersja Kube proxy
+  kubelet-version: Wersja Kubelet
+  memory: Pamięć
+  memory-gib: Pamięć (GiB)
+  memory-usage: 'Zuycie pamięci:'
```
Comment (Wawrzyn321) [inline core/src/i18n/pl.yaml:245]:
```suggestion
  memory-usage: 'Zużycie pamięci:'
```

File: core/src/i18n/pl.yaml [hunk 6]
```diff
@@ -46,38 +51,63 @@ cluster-role-bindings:
...
+    scopes: Zakresy
   list:
     outdated:
-      download: Pobierz Kubeconfig
       tooltip: Nieaktualna wersja parametru może powodować błędy. Usuń i ponownie dodaj swój klaster.
-  overview:
     title: Przegląd klastrów
+  messages:
+    download-failed: Nie udało się pobrać Kubeconfigu
+    error-kubeconfig: Błąd odczytu Kubeconfiga
```
Comment (Wawrzyn321) [inline core/src/i18n/pl.yaml:69]:
```suggestion
    error-kubeconfig: Błąd odczytu Kubeconfigu
```
Chyba?

File: core-ui/src/shared/constants.js [hunk 7]
```diff
@@ -1,9 +1,9 @@
-export const PREFERENCES_TITLE = 'Preferences';
-export const CLUSTERS_OVERVIEW_TITLE = 'Clusters Overview';
-export const ADD_CLUSTER_TITLE = 'Add Cluster';
-export const API_RULES_TITLE = 'API Rules';
-export const NO_PERMISSIONS_TITLE = 'No Permissions';
-export const CLUSTER_OVERVIEW_TITLE = 'Cluster Overview';
```
Comment (Wawrzyn321) [inline core-ui/src/shared/constants.js:6]: Since we're using the translations everywhere, I think we should just move those constants inline.

**Review by Wawrzyn321 — CHANGES_REQUESTED** [review]
See comments. Also, some translations are missing
image

**Review by Wawrzyn321 — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Pull request merged**